src/thrift-client/TgtInterfaceImpl.cpp

```diff
@@ -828,19 +910,33 @@ void StordVmdk::ScheduleNow(folly::EventBase* basep, StorRpcAsyncClient* clientp
 
 	basep->runInEventBaseThread([this, clientp, basep,
```

Comment:
Is batching possible among application IOs and any background IO? if we want so then IO requests has to come from same entry point and that should be outside the thrift recv context. I am referring to the approach where every consumer is adding it's request in a queue and batch algorithm is operating off band on that queue instead of working inband.

Reply:
This comment probably is valid in STORD side code. TGTD does it;s own batching, if any other requester required Bulk interface it has to do it;s own batching.

Doing batching here has advantage that it reduces the RPC overhead. 

Problem with Q and Batch algo is how long to wait for arrival of new requests. I think, this approach is better at TGTD.